cdhweb/static_src/global/components/main-nav-desktop.scss

```diff
@@ -62,6 +62,13 @@
     @include mx.xl {
       width: 160px;
     }
+    // One-off breakpoint window where we want the logo size to be fluid:
+    @media (min-width: fn.em(1616)) {
+      width: 10vw;
```

Comment:
would doing `width: 10vw; max-width: 180px;` work, instead of the `xxxl` rule? I can't access the Jira ticket so I don't know exactly what the issue is.

Reply:
In theory, yes. Let me check that though...